Gemfile

```diff
@@ -6,6 +6,7 @@ source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 ENV['BOLT_DISABLE_ANALYTICS'] = 'true'
 
 gemspec
+gem  "winrm-fs", git: "https://github.com/donoghuc/winrm-fs.git", branch: "BOLT-664"
```

Comment:
Let's try a PR to winrm-fs. It looks like it wasn't quite as easy to separate out, and I don't think we're pressed for time right now.